embedding/csharp/Tizen.Flutter.Embedding/NUIFlutterView.cs

```diff
@@ -0,0 +1,205 @@
+﻿// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using Tizen.NUI;
+using Tizen.NUI.BaseComponents;
+using static Tizen.Flutter.Embedding.Interop;
+
+namespace Tizen.Flutter.Embedding
+{
+    /// <summary>
+    /// Displays a Flutter screen in a Tizen application.
+    /// </summary>
+    public class NUIFlutterView : ImageView
+    {
+        public NUIFlutterView()
+        {
+            _viewSize = Size2D;
+        }
+
+        /// <summary>
+        /// The Flutter engine instance.
+        /// </summary>
+        public FlutterEngine Engine { get; set; } = null;
+
+        /// <summary>
+        /// The Flutter view instance handle.
+        /// </summary>
+        private FlutterDesktopView View { get; set; } = new FlutterDesktopView();
+
+        /// <summary>
+        /// Whether the view is running.
+        /// </summary>
+        public bool IsRunning => !View.IsInvalid;
+
+        /// <summary>
+        /// When the view last received a touch event in milliseconds.
+        /// </summary>
+        private uint _lastTouchEventTime = 0;
+
+        /// <summary>
+        /// The size of the Flutter view.
+        /// </summary>
+        private Size2D _viewSize = new Size2D();
+
+        /// <summary>
+        /// Starts running the view with the associated engine, creating if not set.
+        /// </summary>
+        /// <remarks>
+        /// <see cref="Engine"/> must not be set again after this call.
+        /// <see cref="Destroy"/> must be called if the view is no longer used.
+        /// </remarks>
+        public bool RunEngine()
+        {
+            if (IsRunning)
+            {
+                TizenLog.Error("The engine is already running.");
+                return false;
+            }
+
+            if (Engine == null)
+            {
+                Engine = new FlutterEngine();
+            }
+
+            if (!Engine.IsValid)
+            {
+                TizenLog.Error("Could not create a Flutter engine.");
+                return false;
+            }
+
+            _viewSize = GetDefaultSize();
+
+            Type baseType = typeof(NativeImageQueue).BaseType.BaseType.BaseType;
+            FieldInfo field = baseType.GetField("swigCPtr", BindingFlags.NonPublic | BindingFlags.Instance);
+            NativeImageQueue nativeImageQueue =
+                new NativeImageQueue((uint)_viewSize.Width, (uint)_viewSize.Height, NativeImageQueue.ColorFormat.RGBA8888);
```

Comment:
How about renaming `_viewSize` to just `_size`? Otherwise you need to reformat this line since it exceeds the column length limit (120).

```suggestion
            NativeImageQueue nativeImageQueue = new NativeImageQueue(
                (uint)_viewSize.Width, (uint)_viewSize.Height, NativeImageQueue.ColorFormat.RGBA8888);
```

Reply:
done